Cache.js

```diff
@@ -0,0 +1,39 @@
+const redis = require('redis');
+const defaultHost = '127.0.0.1';
+const defaultPort = 6379;
+const timeToLiveInSeconds = 60;
+
+class Cache {
```

Comment:
One thing you could do here is to remove the knowledge about the client implementation (Redis or Memcache), Cache could receive the client and you could create a test to validate has been consumed checking the behavior, true, false, null, exception.

The creation of the client can be handled by bdk server line 157 or even creating a handler factory